packages/cms-cli/commands/init.js

```diff
@@ -115,6 +159,10 @@ const configureCommanderInitCommand = program => {
   program
     .version(version)
     .description(DESCRIPTION)
+    .option(
```

Comment:
just an fyi, we are removing commander in the very near future https://github.com/HubSpot/hubspot-cms-tools/pull/320

Reply:
Yeah I thought about this, but figured I might as well update them both if they are still there.